frontend/src/hooks/api/useAddRating.ts

```diff
@@ -0,0 +1,9 @@
+import { useMutation } from '@tanstack/react-query';
+import axios from 'axios';
+
+export const useAddRating = () => {
+	return useMutation({
```

Comment:
Shouldn't this also have something like this to invalidate the game query?
```typescript
{
	onSuccess: () => {
		queryClient.invalidateQueries([
			'siteAdmin',
			'organization',
			props.orgId
		]);
	}
}
````
Example stolen from `OrganizationMemberListEntry.tsx`

The same applies to all of the mutaions.